src/machine/board_qtpy_rp2040.go

```diff
@@ -0,0 +1,120 @@
+//go:build qtpy_rp2040
+// +build qtpy_rp2040
+
+package machine
+
+import (
+	"device/rp"
+	"runtime/interrupt"
+)
+
+// Onboard crystal oscillator frequency, in MHz.
+const xoscFreq = 12 // MHz
+
+// GPIO Pins
+const (
+	SDA  = GPIO24
+	SCL  = GPIO25
+	TX   = GPIO20
+	MO   = GPIO3
+	MOSI = GPIO3
+	MI   = GPIO4
+	MISO = GPIO4
+	SCK  = GPIO6
+	RX   = GPIO5
+
+	QT_SCL1 = GPIO23
+	QT_SDA1 = GPIO22
+)
+
+// Analog pins
+const (
+	A0 = GPIO29
+	A1 = GPIO28
+	A2 = GPIO27
+	A3 = GPIO26
+)
+
+const (
+	NEOPIXELS       = GPIO12
+	WS2812          = GPIO12
+	NEOPIXELS_POWER = GPIO11
+	LED             = GPIO20
+)
+
+// I2C Pins.
+const (
+	I2C0_SDA_PIN = GPIO24
+	I2C0_SCL_PIN = GPIO25
+
+	I2C1_SDA_PIN = GPIO26
+	I2C1_SCL_PIN = GPIO27
+
+	I2C1_QT_SDA_PIN = GPIO22
+	I2C1_QT_SCL_PIN = GPIO23
+
+	SDA_PIN = GPIO24
+	SCL_PIN = GPIO25
+)
+
+// SPI default pins
+const (
+	// Default Serial Clock Bus 0 for SPI communications
+	SPI0_SCK_PIN = GPIO6
+	// Default Serial Out Bus 0 for SPI communications
+	SPI0_SDO_PIN = GPIO3 // Tx
+	// Default Serial In Bus 0 for SPI communications
+	SPI0_SDI_PIN = GPIO4 // Rx
+	SPI0_CS      = GPIO5
+
+	// Default Serial Clock Bus 1 for SPI communications
+	SPI1_SCK_PIN = GPIO26
+	// Default Serial Out Bus 1 for SPI communications
+	SPI1_SDO_PIN = GPIO27 // Tx
+	// Default Serial In Bus 1 for SPI communications
+	SPI1_SDI_PIN = GPIO24 // Rx
+	SPI1_CS      = GPIO25
```

Comment:
It is probably better to use a set of GPIO26 to GPIO29 or a set of GPIO25 to GPIO27.
Because the pins are next to each other.

Reply:
oops, You already had the correct setup.
I misread it.

Reply:
Using GPIO 26/27/28/29 would not allow simultaneous use of any ADC pins (they would all be used for SPI1)

The chosen pins are next to each other currently, although the GPIO labeling may seem to indicate they are not.

See 
![image](https://user-images.githubusercontent.com/4591181/188137738-f0c24761-c587-41c8-a053-781f84dc89f7.png)

Reply:
Yes, you are right.